en/docs/administer/directory-structure-of-wso2-products.md

```diff
@@ -7,29 +7,28 @@ folders and files that are specific to the product, in addition to what
+| bin                                           | Contains various scripts ( `            .sh            ` & `            .bat            ` scripts).                                                                                                                                                          | `             <IS_HOME>/bin/            `                           | `             <IS_HOME>/wso2/<PROFILE_HOME>/bin/            `                 |
+| database                                      | Contains the databases.                                                                                                                                                                                                                                        | `             <IS_HOME>/repository/database/            `           | `             <IS_HOME>/wso2/<PROFILE_HOME>/repository/database/            ` |
+| dbscripts                                     | Contains all the database scripts.                                                                                                                                                                                                                             | `             <IS_HOME>/dbscripts/            `                     | `             <IS_HOME>/wso2/<PROFILE_HOME>/dbscripts/            `           |
+| lib                                           | Contains the basic set of libraries required for starting a WSO2 product in standalone mode.                                                                                                                                                                   | `             <IS_HOME>/lib/            `                           | `             <IS_HOME>/wso2/lib/            `                                |
+| repository                                    | The repository where services and modules deployed in a WSO2 product are stored. In addition to this, the `             repository/components            ` directory contains the Carbon runtime and JAR files added by users (such as third party libraries). | `             <IS_HOME>/repository/            `                    | `             <IS_HOME>/wso2/<PROFILE_HOME>/repository/            `          |
+| conf                                          | Contains configuration files.                                                                                                                                                                                                                                  | `             <IS_HOME>/repository/conf/            `               | `             <IS_HOME>/wso2/<PROFILE_HOME>/conf/            `                |
```

Comment:
In IS we don't have profiles..this path might need to change

Reply:
TBD. Will change it in the next PR in case there's a change.